numpy/ma/core.py

```diff
@@ -4356,13 +4361,15 @@ def count(self, axis=None, keepdims=np._NoValue):
                     raise ValueError("'axis' entry is out of bounds")
                 return 1
             elif axis is None:
+                if kwargs.get('keepdims', False):
```

Comment:
The `np._NoValue` is discarded [a few lines earlier](https://github.com/numpy/numpy/blob/master/numpy/ma/core.py#L4341) and the same expression is used [a few lines later](https://github.com/numpy/numpy/blob/master/numpy/ma/core.py#L4371).

It's a bit weird to use `kwargs` if the only `kwarg` is `keepdims` but I _guess_ that's more easily extendable if one ever wants to allow additional `kwargs`. :sweat_smile: